app/models/piece.rb

```diff
@@ -7,11 +7,39 @@ def vectors(vectors = [])
     @vectors ||= vectors
   end
 
-  def move(x, y)
-    if self.move_capable?(position)
-      @x = x
-      @y = y
+  def move(params)
+    xpos = params[:x].to_i
+    ypos = params[:y].to_i
+    channel = "public-conversation"
+    if game.piece_at?(xpos, ypos)
+      captured_piece = game.piece_at(xpos, ypos)
+      captured_piece_fig = captured_piece.figure
+      captured_piece.update_attributes(x: 8, y:8, state: 'off')
+      Pusher.trigger(channel, 'move_event', {
+        piece: captured_piece,
+        fig: captured_piece_fig
+      })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.